lte/protos/session_manager.proto

```diff
@@ -951,18 +1017,26 @@ message UPFSessionConfigState {
      repeated UPFSessionState upf_session_state = 1;
 }
 
+message UPFPagingInfo {
+  string subscriber_id = 1;
+  uint32 local_f_teid = 2;
+   string ue_ip_addr = 3;
+}
 //Set Interface from Pipelined
 service SetInterfaceForUserPlane {
-    rpc SetUPFNodeState(UPFNodeState) returns (magma.orc8r.Void) {}
-    rpc SetUPFSessionsConfig(UPFSessionConfigState) returns (magma.orc8r.Void) {}
+    rpc SetUPFNodeState(UPFNodeState) returns (SmContextVoid) {}
+    rpc SetUPFSessionsConfig(UPFSessionConfigState) returns (SmContextVoid) {}
+    rpc SendPagingRequest(UPFPagingInfo) returns (SmContextVoid) {}
 }
 
-// RPC service and method
+//SET API for 3 procedures request from AMF to SMF
+service AmfPduSessionSmContext {
+  rpc SetAmfSessionContext(SetSMSessionContext) returns (SmContextVoid) {}
+  rpc SetSmfNotification(SetSmNotificationContext) returns (SmContextVoid) {}    //AMF to SMF
+}
+// SET API RPC service and method request from SMF to AMF
 service SmfPduSessionSmContext {
  rpc SetAmfNotification(SetSmNotificationContext) returns (SmContextVoid){}    //SMF to AMF
- rpc SetSmfSessionContext(SetSMSessionContextAccess) returns (SmContextVoid);
+ rpc SetSmfSessionContext(SetSMSessionContextAccess) returns (SmContextVoid) {}
```

Comment:
This is not an SMS service, but an AMF service. Why is this defined here?

Reply:
Not sure, AMF team has to answer ?

Reply:
As such mme.proto or amf.proto defined,   so we kept it here.

Reply:
AMF team plan to raise separate PR to track this,  they will remove once they create on another proto.